File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-a-deleter-method_1.cs
        public static void Delete(int salesOrderID, int salesOrderDetailID)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");

            SalesOrderDetail SalesOrderDetail =
                   (from SalesOrderDetails in dataContext.SalesOrderDetails.AsEnumerable().Take(20)
                    where SalesOrderDetails.SalesOrderID == salesOrderID &&
                    SalesOrderDetails.SalesOrderDetailID == salesOrderDetailID
                    select SalesOrderDetails).Single();

            dataContext.SalesOrderDetails.DeleteOnSubmit(SalesOrderDetail);
            dataContext.SubmitChanges();
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-a-deleter-method_1.vb
    Public Shared Sub Delete(ByVal salesOrderID As Integer, ByVal salesOrderDetailID As Integer)
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim SalesOrderDetail As SalesOrderDetail = _
            (From SalesOrderDetails In dataContext.SalesOrderDetails.AsEnumerable().Take(20) _
            Where SalesOrderDetails.SalesOrderID = salesOrderID And _
                  SalesOrderDetails.SalesOrderDetailID = salesOrderDetailID _
            Select SalesOrderDetails).Single()

        dataContext.SalesOrderDetails.DeleteOnSubmit(SalesOrderDetail)
        dataContext.SubmitChanges()

    End Sub